packages/GT/GT.0.5.4/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "2.0"
+synopsis: "Generic programming with extensible transformations"
+description: """
+Yet another library for generic programming. Provides syntax extensions
+both for camlp5 and PPX which allow decoration of type declarations with
+following compile-time code generation. Provides the way for creating
+plugins (compiled separately from the library) for enchancing supported
+type transformations.
+
+Strongly reminds the `visitors` library from François Pottier.
+During desing of a library of these kind there many possible
+design decision and in many cases we decided to implement
+the decision opposite to the one used in `visitors`.
+
+
+P.S. Since 2023 development team is no longer associated with JetBrains Research"""
+maintainer: ["Kakadu@pm.me"]
+authors: ["https://github.com/dboulytchev" "https://github.com/Kakadu"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/PLTools/GT"
+bug-reports: "https://github.com/PLTools/GT/issues"
+depends: [
+  "ocaml" {>= "4.14" & < "5.0.0" | >= "5.3.0" & < "5.4.0"}
+  "dune" {>= "3.16" & >= "3.16"}
```

Comment:
```suggestion
  "dune" {>= "3.16"}
```
This can be avoided by removing dune from the dependencies (it adds itself automatically)